Scripts/DEADataHandling.py

```diff
@@ -172,7 +159,8 @@ def load_sentinel(dc, product, query, filter_cloud=True, **bands_of_interest):
         print('loaded {}'.format(product))
         if filter_cloud:
             print('making mask')
-            clear_pixels = np.logical_and(ds.pixel_quality != 2, ds.pixel_quality != 3)
+            clear_pixels = np.logical_and(ds.pixel_quality != 0, ds.pixel_quality != 2, 
+                                          ds.pixel_quality != 3)
```

Comment:
This doesn't explain what is being masked out.
See https://nbviewer.jupyter.org/github/opendatacube/datacube-core/blob/develop/examples/notebooks/Datacube_Summary.ipynb#Masking-out-cloud

Reply:
probably because I don't understand the sentinel masking